app/lib/seeder.rb

```diff
@@ -25,9 +25,27 @@ def self.load_fraud_indicators
     end
   end
 
+  def faqify!
```

Comment:
if we are adding this should we remove the `StateFile::FaqDatabaseExportService`, `app/services/state_file/faq_database_export_en.yml` and `app/services/state_file/faq_database_export_es.yml` as well?